### PR TITLE
user varargs in store the watch callback

### PR DESCRIPTION
**Pull request by vieux**
carry changes in #772

### CODE REVIEW

**Comment by vieux** [comment]
ping @abronan @mavenugo

File: pkg/store/zookeeper.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func (s *Zookeeper) Watch(key string, _ time.Duration, callback WatchCallback) e
 			entry, index, err := s.Get(key)
 			kvi := []KVEntry{&kviTuple{key, []byte(entry), index}}
 			if err == nil {
-				callback(kvi)
```
Comment (mavenugo) [inline pkg/store/zookeeper.go]: Minor comment. It would be cool to remove the array here as well (just like you did for consul and etcd). 
otherwise LGTM.

Comment (vieux) [inline pkg/store/zookeeper.go]: oh yes sorry

**Comment by abronan** [comment]
LGTM

**Pull request merged**